tests/jointcalTestBase.py

```diff
@@ -75,6 +75,8 @@ def setUp_base(self, center, radius,
             tests are running. See the developer docs about logging for valid
             levels: https://developer.lsst.io/coding/logging.html
         """
+        self.path = os.path.dirname(__file__)
```

Comment:
Minor point, but the types in the docstrings above should have backticks. Also in `_testJointcalTask` below.

Reply:
Ah the joys of old docstrings.